Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
Can you elaborate on why `new` and `create` need to have different behaviour? Will you make `new` private? If not, the API seems pretty confusing.